tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/legacy@2019_03_12/stacktrace_group_modules_1.pysnap

```diff
@@ -0,0 +1,39 @@
+---
+created: '2021-02-09T12:19:08.406694Z'
+creator: sentry
+source: tests/sentry/grouping/test_variants.py
+---
+app:
+  hash: null
+  component:
+    app
+      stacktrace
+        frame (frame considered in-app because no frame is in-app)
+        frame (ignored due to recursion)
```

Comment:
Any idea why it's marking this frame as recursion? The instruction address is different in every frame.

Reply:
The instruction addr is not considered for the recursion flag:

https://github.com/getsentry/sentry/blob/bcafccd903f78f18b866738c5993bd31032aa20c/src/sentry/grouping/strategies/newstyle.py#L41-L63

Reply:
Oh, on native that's actually broken then.